### PR TITLE
Extend `clean-conversations` to new comment forms

### PR DESCRIPTION
**Pull request by kidonng**
Cherry-picked from [`84c9dc1` (#5113)](https://github.com/refined-github/refined-github/pull/5113/commits/84c9dc1f747c144ddf22accfce531e7a00c7fafb)

Mentioned in #1066

## Test URLs

- Conversations: Here
- Discussions: https://github.com/vercel/vercel/discussions/3874
- Commits: https://github.com/refined-github/refined-github/commit/82fb3d62f11838ad4120f510bd90520c57bb12da

## Screenshot

<table>
<tr>
	<td align="center"><strong>Before</strong>
	<td align="center"><strong>After</strong>
<tr>
	<td>
	<td>
</table>

### CODE REVIEW

**Review by fregante — DISMISSED** [review]

File: source/features/clean-conversations.css [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+:is(.review-thread-reply, #issuecomment-new) .avatar {
 	display: none;
 }
 
 .review-thread-reply .review-thread-reply-button {
 	margin-left: 0;
 }
+
+/* Hide arrow on new comment form */
+.timeline-comment--caret::before,
```
Comment (fregante) [inline source/features/clean-conversations.css:11]: Does this really only  target the form? The selector is generic

File: source/refined-github.css [hunk 2]
```diff
@@ -35,15 +35,6 @@
...
 
-/* Remove user avatar from new comment form on threads */
-.review-thread-reply .avatar {
-	display: none;
-}
-
-.review-thread-reply .review-thread-reply-button {
-	margin-left: 0;
-}
-
```
Comment (kidonng) [inline source/refined-github.css:46]: Apparently forgotten in [`f099d37` (#1986)](https://github.com/refined-github/refined-github/pull/1986/commits/f099d3737111e9d04917267f9ebb2a4efc057368)

File: source/features/clean-conversations.css (hunk 1 above)
Comment (kidonng) [reply source/features/clean-conversations.css]: Good catch, this was done in a hurry in my previous lint PR 😅

File: source/features/clean-conversations.css [hunk 3]
```diff
@@ -1,8 +1,14 @@
-/* Remove user avatar from new comment form on threads */
-.review-thread-reply .avatar {
+/* Remove user avatar from new comment form */
+:is(.review-thread-reply, .timeline-new-comment) .avatar {
```
Comment (kidonng) [inline source/features/clean-conversations.css:2]: - Despite the name, `.review-thread-reply` is also used in Discussions thread reply (could it be that they are review comments in disguise? 🤔)
- `.timeline-new-comment` works on both conversations and commits page

**Review by fregante — APPROVED** [review]
Untested but LGTM

**Pull request merged**